scripts/eod_1_otc_services_dict.py

```diff
@@ -354,15 +379,17 @@ def run():
 
     logging.info("-------------------------OTC SERVICES DICT SCRIPT IS RUNNING-------------------------")
 
-    BASE_DIR_SWISS = "/repos/infra/otc-metadata-swiss/contents/"
-    BASE_DIR_REGULAR = "/repos/infra/otc-metadata/contents/"
+    BASE_DIR_UNIFIED = "/repos/infra/otc-metadata-rework/contents/"
     STYRING_URL_REGULAR = "/repos/infra/gitstyring/contents/data/github/orgs/opentelekomcloud-docs/data.yaml?token="
     STYRING_URL_SWISS = "/repos/infra/gitstyring/contents/data/github/orgs/opentelekomcloud-docs-swiss/data.yaml?token="
     BASE_RTC_TABLE = "repo_title_category"
     BASE_DOC_TABLE = "doc_types"
 
-    main(BASE_DIR_REGULAR, BASE_RTC_TABLE, BASE_DOC_TABLE, STYRING_URL_REGULAR)
-    main(BASE_DIR_SWISS, f"{BASE_RTC_TABLE}_swiss", f"{BASE_DOC_TABLE}_swiss", STYRING_URL_SWISS)
+    main(BASE_DIR_UNIFIED, BASE_RTC_TABLE, BASE_DOC_TABLE, STYRING_URL_REGULAR, target_cloud="eu_de")
```

Comment:
instead of hardcoding can you dynamically query it from otc-metadata to avoid that in future you need to change it with every new env?

Reply:
no, if you change target_cloud=TARGET_CLOUD, you can loop TARGET_CLOUD from fetched values from otc-metadata-rework fetch, in such way you always dynamically refer to values  which are defined in otc-metadata